drivers/hmis/lib/tasks/ce_starter_pack_20250302.rake

```diff
@@ -48,7 +45,139 @@ task ce_starter_pack_20250302: [:environment] do
+  form_definition.version ||= 0
+  form_definition.definition = {
+    "item": [
+      {
+        "text": "Date Contacted",
+        "type": "DATE",
+        "link_id": "date_contacted",
+        "required": true,
+        "read_only": false,
+        "warn_if_empty": false,
+        "disabled_display": "HIDDEN",
+        "mapping": {"custom_field_key": "ce_one_task_date_contacted"}
+      },
+      {
+        "text": "Client Accepts Referral",
+        "type": "CHOICE",
+        "link_id": "client_accepted",
+        "required": true,
+        "read_only": false,
+        "warn_if_empty": false,
+        "disabled_display": "HIDDEN",
+        "pick_list_options": [
+          {
+            "code": "1",
+            "label": "Yes, client accepts referral"
+          },
+          {
+            "code": "0",
+            "label": "No, client does not accept referral or could not be contacted"
+          }
+        ],
+        "mapping": {"custom_field_key": "ce_one_task_client_accepted"}
+      },
+      {
+        "text": "If you submit, the referral will be stopped and sent to an Admin for review. This cannot be undone.",
+        "type": "DISPLAY",
+        "link_id": "stop_referral_if_you_submit_the_referral",
+        "component": "ALERT_ERROR",
+        "enable_when": [
+          {
+            "operator": "EQUAL",
+            "question": "client_accepted",
+            "answer_code": "0"
+          }
+        ],
+        "enable_behavior": "ALL",
+        "disabled_display": "HIDDEN"
+      },
+      {
+        "text": "Notes",
+        "type": "TEXT",
+        "link_id": "notes",
+        "disabled_display": "HIDDEN",
+        "mapping": {"custom_field_key": "ce_one_task_notes"}
+      }
+    ]
+  }
+  form_definition.save! if form_definition.changed?
+  # TODO - add CDEDs? Not necessary/used for CE, but the form builder prevents edits if it can't find them
+
+  client_acceptance_task = Hmis::WorkflowDefinition::Task.find_or_initialize_by(
+    form_definition_id: form_definition.id
+  )
+  client_acceptance_task.name = 'Confirm Client Accepts Referral'
+  client_acceptance_task.template = one_task_template
+  client_acceptance_task.swimlane = case_managers
+
+  gateway = Hmis::WorkflowDefinition::Gateway.find_or_initialize_by(
+    template: one_task_template,
+    gateway_type: 'exclusive',
+    name: 'client acceptance'
+  )
+  start_workflow_event.connect_to!(client_acceptance_task) unless start_workflow_event.outflows.where(target_node_id: client_acceptance_task.id).exists?
+  client_acceptance_task.connect_to!(gateway) unless client_acceptance_task.outflows.where(target_node_id: gateway.id).exists?
+  gateway.connect_to!(reject_workflow_event, condition: 'client_accepted = 0') unless gateway.outflows.where(target_node_id: reject_workflow_event.id).exists?
+  gateway.connect_to!(accept_workflow_event, condition: 'client_accepted = 1') unless gateway.outflows.where(target_node_id: accept_workflow_event.id).exists?
+
```

Comment:
This comment is for line 223 below
```
clients = Hmis::Hud::Client.all.limit(100) # modify this if you want to include different or specific clients
```
Suggest changing to `Hmis::Hud::Client.hmis.limit(100)`  to avoid issues with trying to resolve clients from different data sources, which causes an error in the frontend:



<img width="1200" alt="Screenshot 2025-03-13 at 8 32 04 AM" src="https://github.com/user-attachments/assets/c13d07b9-2cf7-407c-bda3-a3774b39c3a0" />